src/api/createVoiceVideoCalling.ts

```diff
@@ -340,11 +340,15 @@ export class VoiceVideoCallingProxy {
                 SecondaryChannelEventType: SecondaryChannelEvents.End
             }
 
+            this.scenarioMarker?.startScenario(TelemetryEvent.onCallDisconnected);
+
             try {
                 this.callingParams?.OCClient.makeSecondaryChannelEventRequest(this.callingParams?.chatToken.requestId, body);
                 this.debug && console.debug(`[VoiceVideoCallingProxy][onCallDisconnected][makeSecondaryChannelEventRequest]`);
+                this.scenarioMarker?.completeScenario(TelemetryEvent.onCallDisconnected);
             } catch (e) {
                 console.error(`[VoiceVideoCallingProxy][onCallDisconnected][makeSecondaryChannelEventRequest] Failure ${e}`);
+                this.scenarioMarker?.failScenario(TelemetryEvent.onCallDisconnected);
```

Comment:
We need to pass `callId` & `ExceptionDetails`. You can refer to [this](https://github.com/microsoft/omnichannel-chat-sdk/blob/f9059fb6eec2623470efb30fce52e630a4a7d9dc/src/api/createVoiceVideoCalling.ts#L170)